addins/ShortcutSample/ShortcutSample.xml

```diff
@@ -0,0 +1,154 @@
+<?xml version="1.0" encoding="UTF-8"?>

```

Comment:
In general, I suggest cloning one of the samples we have here, and removing the parts that you don't need (as opposed to cloning an internal test add-in).